source/configuration_manual/virtual_plugin.rst

```diff
@@ -35,6 +35,21 @@ default it uses the `fs` layout, so you can create directories such as:
 If you prefer to use Maildir++ layout instead, you can simply append
 ``:LAYOUT=maildir++`` to the location.
 
+Storage location with obox
+--------------------------
+
+When using the virtual plugin with obox, the virtual indexes folder must be
+configured to be placed in the users home directory under the foldername
+"virtual" otherwise the indexes will be lost when cleaning metacache.
+
+.. code-block:: none
+
+        location = virtual:/etc/dovecot/virtual:INDEX=~/virtual
```

Comment:
and this too.

Reply:
Here it's correct i think as it's in the virtual plugin not the virtual-attachments.